keramik/src/monitoring.md

```diff
@@ -18,6 +18,7 @@ spec:
   replicas: 2
   monitoring:
     namespaced: true
+    podMonitor: true
```

Comment:
```suggestion
    podMonitoring:
      enabled: true
```

Reply:
A bit of a mixup, these should ALL be `podmonitor` and no `podmonitoring` at all.

Confusingly, `podmonitor` is part of an upstream prometheus operator project that a few projects use and the Grafana Cloud agent supports.
https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.PodMonitor

`podmonitoring` is a GCP provided, prometheus compatible, cluster provided, CRD that we where thinking about using.
https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/doc/api.md#monitoring.googleapis.com/v1.PodMonitoring

Reply:
Thanks.. that mostly makes sense 😅 I was trying to update the docs to match the spec so it actually succeeds, but it didn't do anything since the crd had to be installed. I'm good with whatever you guys decide, but if you could update the docs with how (if) it's something that should be done locally, that'd be awesome.

Reply:
Yeah, agreed.

@3benbox, would you mind telling me how the CRD normally gets created 😅? I can add it to the README.

Reply:
Hey @dav1do, I'll add this to the README once I get a chance to chat with Ben. I'm going to merge the PR because we're stuck in our testing without pod monitoring and we don't know how to create them quickly by hand 🙈